src/sequential/utils.rs

```diff
@@ -26,6 +26,17 @@ use rust_sodium;
 use std::cmp;
 use std::io::Write;
 use std::sync::{ONCE_INIT, Once};
+use tiny_keccak::Keccak;
+
+pub const HASH_SIZE: usize = 32;
+
+pub fn sha3_256_hash(input: &[u8]) -> Vec<u8> {
```

Comment:
we don't need this - we already have a function `sha3_256` in `tiny_keccak` so pls use that.